api/users/serializers.py

```diff
@@ -211,7 +211,7 @@ def get_institutions_count(self, obj):
 
     def get_can_view_reviews(self, obj):
         group_qs = AbstractProviderGroupObjectPermission.objects.filter(group__user=obj, permission__codename='view_submissions')
-        return group_qs.exists() or obj.abstractprovideruserobjectpermission_set.filter(permission__codename='view_submissions')
+        return group_qs.exists() or obj.abstractprovideruserobjectpermission_set.filter(permission__codename='view_submissions') or []
```

Comment:
if we don't do this the return value is a QuerySet which won't serialize [at this line](https://github.com/CenterForOpenScience/osf.io/pull/9371/files#diff-c1703080e33331cb3e1bfb4f1ddfeb01R276)